org-agenda-conflict.el

```diff
@@ -27,6 +27,9 @@
 
 ;; This code mark conflicting items in the org agenda. Conflicting
 ;; items are items with an overlap between their start and end date.
+;; The marking assumes that items are sorted and only checks for
```

Comment:
This is no longer true since the new checking compare line 1 with line 2, line 3, line 4... (on the same day), then line 2 with line 3, 4, etc.

Reply:
Removed this.